docs/en/test_and_ci/fuzz_tests.md

```diff
@@ -0,0 +1,69 @@
+# Fuzz Tests
```

Comment:
> there is also some docs on the unit tests page, and I wasn't sure if it deserves its own page.

@bkueng There are good arguments either way. I decided it was worth doing because the tests are written differently. As long as they are cross-linked and we make it clear this is a type of unit test I think this works better.

Note, I did change the text to note that fuzz tests are specialised (not generalized) unit tests. Otherwise most of the doc is the same except for running prettier and cross linking.

Reply:
No, they are generalized :)
You could turn any unit test into a fuzz test. Fuzz tests are a superset in that they allow arguments as inputs.